codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Stream.java

```diff
@@ -158,6 +158,18 @@
     Http2Stream parent();
 
     /**
+     * Returns {@code true} if this node is viable in its own right to be in the priority tree and {@code false}
+     * otherwise. For example if {@link #state()} returns {@link State#CLOSED} then this node is not considered viable.
+     * Once a {@link Http2Stream} becomes inviable it will not transition back to viable.
+     */
+    boolean isViable();
```

Comment:
> Hmm ... I don't like viable :/

+1.  My goal in life is to introduce a "clear" concept and then let the word smiths figure out the rest :)